libr/anal/p/anal_x86_cs.c

```diff
@@ -1136,7 +1136,15 @@ static void anop_esil (RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len
 		{
 			char *src = getarg (&gop, 1, 0, NULL);
 			char *dst = getarg (&gop, 0, 1, "&");
+			char *p;
 			esilprintf (op, "%s,%s,$0,of,=,$0,cf,=,$z,zf,=,$s,sf,=,$o,pf,=", src, dst);
+			if ((a->bits == 64) && (p = strchr (dst, (int)','))) {
+				char *dst64;
```

Comment:
because I wanted a meaningful name for the 64-bit register string. I thought p for pointer is meaningful and dst64 for 64-bit destination is meaningful